app/controllers/feedbacks_controller.rb

```diff
@@ -61,6 +62,19 @@ def set_feedback
     authorize @feedback
   end
 
+  def create_redirect
+    if session[:request_url].include?('last_submitted_hours_and_feedbacks')
```

Comment:
I'd suggest to simplify this by having it just to return the path, and not the `redirect_to`. So:
```ruby
def create
  # ...
  redirect_to create_redirect, make_notice
```